articles/quickstart/webapp/aspnet-owin/interactive.md

```diff
@@ -27,7 +27,7 @@ Auth0 allows you to quickly add authentication and gain access to user profile i
   returnTo: 'http://localhost:3000'
 }) %>
 
-## Install and Configure the Middleware {{{ data-action=code data-code="Startup.cs#9:13" }}}
+## Setup the Project {{{ data-action=code data-code="Web.config#1:7" }}}
```

Comment:
I believe Setup the Project sounds like we are helping them set up the entire project, while all were are doing is help them install and configure the middleware.